app/src/main/java/com/bignerdranch/android/geoquiz/CheatActivity.kt

```diff
@@ -51,21 +45,22 @@ class CheatActivity : AppCompatActivity() {
         }
     }
 
-    /**
-     * This is my companion object
-     */
-    companion object {
-        fun newIntent(packageContext: Context, answerIsTrue: Boolean): Intent {
-            return Intent (packageContext, CheatActivity::class.java).apply {
-                putExtra(EXTRA_ANSWER_IS_TRUE, answerIsTrue)
-            }
-        }
-    }
-
     private fun setAnswerShownResult(isAnswerShown: Boolean) {
         val data = Intent().apply {
             putExtra(EXTRA_ANSWER_SHOWN, isAnswerShown)
         }
         setResult(Activity.RESULT_OK, data)
     }
+
+    companion object {
+
+        const val EXTRA_ANSWER_SHOWN = "com.bignerdranch.android.geoquiz.answer_shown"
```

Comment:
Can't this be private? Are you using it outside of this class?

Reply:
Yes, it is used in `MainActivity` class